TikTokApi/tiktok.py

```diff
@@ -129,15 +137,16 @@ def getData(self, **kwargs) -> dict:
         if self.request_delay is not None:
             time.sleep(self.request_delay)
 
-        if self.proxy != None:
+        if self.proxy is not None:
             proxy = self.proxy
 
-        if self.signer_url == None:
+        if self.signer_url is None:
             verify_fp, did, signature = self.browser.sign_url(**kwargs)
             userAgent = self.browser.userAgent
             referrer = self.browser.referrer
         else:
-            verify_fp, did, signature, userAgent, referrer = self.external_signer(kwargs['url'], custom_did=kwargs.get('custom_did', None))
+            verify_fp, did, signature, userAgent, referrer = self.external_signer(
+                kwargs['url'], custom_did=kwargs.get('custom_did', None))
```

Comment:
another unneecessary `None` default here.

Reply:
I see, I think the nightly branch has been merged in this PR. this explain all theese changes
Thanks for you code review